src/builtins/ThresholdPredictors.jl

```diff
@@ -138,7 +138,7 @@ function MMI.fit(model::ThresholdUnion, verbosity::Int, args...)
         end
     end
     model_fitresult, model_cache, model_report = MMI.fit(
-        model.model, verbosity-1, args...
+        model.model, verbosity-1, map(unwrap_target, args)...
     )
```

Comment:
`unwrap_target` seems like a confusing name choice - you are not just calling this on a target variable, but an all the data args. Maybe `unwrap` is better.